cyhy_report/customer/generate_report.py

```diff
@@ -3432,6 +3437,15 @@ def __generate_mustache_json(self, filename):
             t["first_detected_time_tex"] = t["first_detected"].strftime("{%H}{%M}{%S}")
             t["last_detected_date_tex"] = t["last_detected"].strftime("{%d}{%m}{%Y}")
             t["last_detected_time_tex"] = t["last_detected"].strftime("{%H}{%M}{%S}")
+            # Trim long descriptions so that they don't overflow the page
+            # (LaTeX table cells cannot span pages, as far as I know) or cause
+            # other issues. For more information, see:
+            # - https://github.com/cisagov/cyhy-reports/issues/124
+            # - https://github.com/cisagov/cyhy-reports/issues/123
```

Comment:
Just an ordering preference.

```suggestion
            # - https://github.com/cisagov/cyhy-reports/issues/123
            # - https://github.com/cisagov/cyhy-reports/issues/124
```

Reply:
I ordered them like that because #124 was more directly relevant to this PR while #123 was being resolved as a side effect, but I agree that it looks better to have them ordered numerically, so I'll make this change.